dashboard/app/controllers/courses_controller.rb

```diff
@@ -3,9 +3,16 @@ class CoursesController < ApplicationController
 
   before_action :require_levelbuilder_mode, except: [:index, :show, :vocab, :resources, :code, :standards]
   before_action :authenticate_user!, except: [:index, :show, :vocab, :resources, :code, :standards]
+  before_action :set_unit_group, only: [:show, :vocab, :resources, :code, :standards, :edit, :update, :get_rollup_resources]
   before_action :set_redirect_override, only: [:show]
   authorize_resource class: 'UnitGroup', except: [:index]
 
+  def set_unit_group
+    @unit_group = UnitGroup.get_from_cache(params[:course_name])
+    # the standards and show actions allow redirecting when given a family name so we do not want to raise in that case
+    raise ActiveRecord::RecordNotFound unless @unit_group || (['standards', 'show'].include?(params[:action]) && UnitGroup.family_names.include?(params[:course_name]))
```

Comment:
Thanks for the comment here. Just to be clear: do both 'standards' and 'show' need to be included in the actions? Is there a case when only one would be present?

Reply:
Its just saying if the action is either standards or show we will not raise if the param is a family name instead of a unit group

Reply:
Got it, thanks 👍